.github/workflows/unix.yml

```
@@ -1,13 +1,18 @@
 name: Unix
 
-on: [push, pull_request]
+on:
+  push:
+  pull_request:
+  # Run daily at midnight (UTC).
+  schedule:
+    - cron: '0 0 * * *'
 
 jobs:
   test:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: [3.8]
+        python-version: [3.8, 3.9, '3.10', 3.11]
```

Not sure of the syntax, but do the single quote actually do sth or should this be the same for all versions?
```suggestion
        python-version: [3.8, 3.9, 3.10, 3.11]
```

---

Yep, the quotes make it a string. Otherwise YAML interprets it as a double and will read `3.1` instead of `3.10`.